packages/ui/src/council/queries/council.graphql

```diff
@@ -87,6 +87,11 @@ fragment ElectionRoundFields on ElectionRound {
   }
 }
 
+fragment LatestElectionRoundFields on ElectionRound {
+  ...ElectionRoundFields
+  isFinished
+}
```

Comment:
Looks like this fragment is not used

Reply:
It's used for `LatestElectionRoundFieldsFragment` in `council/types/LatestElection.ts`, but it should be also applied in `query GetLatestElection`, fixed it :)